<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-4GSVHLM" failures="1" tests="2" name="Car Loan" time="0.089" errors="0" timestamp="2023-10-01T18:59:10 IST">
  <testcase classname="test.Day4" name="APILoginCarLoan" time="0.037">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day4.APILoginCarLoan(Day4.java:38)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- APILoginCarLoan -->
  <testcase classname="test.Day4" name="dependencyWith" time="0.0">
    <skipped/>
  </testcase> <!-- dependencyWith -->
  <testcase classname="test.Day4" name="MobileSignininCarLoan" time="0.0">
    <ignored/>
  </testcase> <!-- MobileSignininCarLoan -->
  <testcase classname="test.Day4" name="webLogincarLoan" time="0.0">
    <ignored/>
  </testcase> <!-- webLogincarLoan -->
  <testcase classname="test.Day4" name="MobileLoginCarLoan" time="0.0">
    <ignored/>
  </testcase> <!-- MobileLoginCarLoan -->
</testsuite> <!-- Car Loan -->
